client/client-multi/client-c/test/test_kaa_bootstrap.c

```diff
@@ -15,6 +15,10 @@
  */
 
 #include <string.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include <stdlib.h>
```

Comment:
Because without this include its not compile.
We remove all system dependent definitions from headers, so they should be in every compiled C files.